action.yml

```diff
@@ -28,6 +28,9 @@ inputs:
   app_build_command:
     description: "Custom command for Oryx to run when building application source code"
     required: false
+  is_static_export:
```

Comment:
Can you please pull down the latest changes from Azure:v1, just so we can see exactly what the resulting change will look like once merged?